### DISCUSSION TITLE
[Announcement] SoC Honor Code 2023

### DISCUSSION THREAD

**Discussion by KihongHeo** [prosyslab-classroom/cs348-information-security#383] [Announcements]
Hi all,

I think all of you have received SoC Honor code 2023 from the department. Please read carefully and sign the document until next Fri. Only the students who sign the document will be considered for grading.

No sign, no gain!

**Comment by bongjunj** [comment]
Dear Prof., I can't find the honor code in mailbox, portal notices, and others. Could you locate where I can find that?

**Comment by KihongHeo** [comment]
Please check your junk mailbox. Or ask the department office. Sorry I also don't have the link. It was sent to only students as I know.

**Comment by ICubE01** [comment]
I asked SoC staff, and he said that he'll send a mail when drop period is done.

**Comment by KihongHeo** [comment]
@ICubE- Thanks.

**Comment by Re-st** [comment]
Dear all, 
Is there a portal link or sth that links to SoC Honor code?
I haven't received Email yet in Thursday, unlike other semesters. 
I am curious if this is happening to many people.

**Comment by tanapthetimid** [comment]
I haven't received anything either...

**Comment by KihongHeo** [comment]
Maybe the admin office needs some more time. No worries.

**Comment by tanapthetimid** [comment]
I emailed the person who sent out the forms last semester and he said there is no such form and that the only honor code is the one shown in academic system. I am confused.

**Comment by castle-bell** [comment]
I know today is the due date, but I haven't received an email yet. Could you extend the deadline?

**Comment by KihongHeo** [comment]
Sure. No worries. Let me announce again later.

**Comment by rzhsynv** [comment]
The staff just sent the mail.

**Comment by KihongHeo** [comment]
Good. See https://forms.gle/mxApoGi1vMkcCJ7n8.
Due: Mar 15
